src/Tools/XmlDocumentationsUtility/Program.cs

```diff
@@ -183,19 +187,28 @@ internal static void RemoveDocumentationForHiddenNodes(string xmlPath,ZeroTouchM
 
```

Comment:
check of Properties.Resources outside switch statement if module==null and then continue. If module is null then rest of the checks won't make sense, so no point going into switch block.